lib/holidays/finder/rules/year_range.rb

```diff
@@ -0,0 +1,82 @@
+# Please note that only one condition needs to match in order for `call` to return `true.
+# See the test file for this class for specific examples.
+module Holidays
+  module Finder
+    module Rules
+      class YearRange
+        class << self
+          BEFORE = :before
+          AFTER = :after
+          LIMITED = :limited
+          BETWEEN = :between
+
+          #TODO Can we just accept symbols here? Why accept strings?
```

Comment:
If we use load_custom method, operator return string.So i added string operator.
If AFTER.to_s is not written, following case may occur.
`irb(main):001:0> Holidays.load_custom('definitions/jp.yaml')
irb(main):002:0> Holidays.on(Date.civil(2016,8,11), :jp)
=> []`

I have found above case when i was testing test_custom_year_range_holidays.rb.